src/applications/letters/containers/LettersApp.jsx

```diff
@@ -58,13 +59,12 @@ export class LettersApp extends React.Component {
         verify
         serviceRequired="evss-claims"
         user={this.props.user}>
-        <DowntimeNotification appTitle="Letters Generator" dependencies={[services.evss]}>
-          <AppContent>
-            <div>
-              {this.props.children}
-            </div>
-          </AppContent>
-        </DowntimeNotification>
+        <AppContent>
+          <DowntimeBanner appTitle="Letters Generator" dependencies={[services.evss]}/>
```

Comment:
Do we know what this `services.evss` is hooked up to? My understanding is EVSS provides a whole gamut of services. This definitely isn't letter-generator-specific, but I'm wondering if there's a case in which this would flag EVSS as being down while the letter generator is actually still up.

cc @kreek @bshyong @ncksllvn @jbalboni because I don't know who would know that answer.

Reply:
I don't think it's necessarily tied directly to any EVSS systems - when we become aware of an outage, we schedule it ourselves under that service name.

Reply:
I think the concern is that if we use this `evss` service for multiple apps (claim status also uses evss, for example), we might end up putting apps in maintenance that aren't actually down, depending on what's actually happening. I don't know enough about EVSS to say if it normally goes down all at once or if we can have some services up, but others be down.

Reply:
Ahh, yeah. I share that same concern with @jbalboni. If that's the case, do you know what the level of effort is to make more granular services?